.goreleaser.yml

```diff
@@ -91,10 +91,14 @@ checksum:
   name_template: '{{ .ProjectName }}_v{{ .Version }}_SHA256SUMS'
   algorithm: sha256
 signs:
-  - cmd: signore
-    args: ["sign", "--dearmor", "--file", "${artifact}", "--out", "${signature}"]
-    artifacts: checksum
-    signature: ${artifact}.sig
-
+- artifacts: checksum
+  args:
+  - “--batch”
+  - “--local-user”
+  - “{{ .Env.GPG_FINGERPRINT  }}”
+  - “--output”
+  - “${signature}”
+  - “--detach-sign”
+  - “${artifact}”
```

Comment:
Are these quotes considered as quotes or characters ?

Reply:
Ah good catch. This is a copy paste issue for sure. I will update now.